README.md

```diff
@@ -98,7 +98,8 @@ sudo apt-get install ponyc-master
 
 ## Windows using ZIP (via Bintray)
 
-For Windows, the `master` and `release` branches are packaged and availabe on Bintray ([pony-language/ponyc-win](https://bintray.com/pony-language/ponyc-win)):
+For Windows, the `master` and `release` branches are packaged and 
```

Comment:
Looks like you accidentally inserted a line break here and removed the word "availabe" instead of fixing the spelling of it.